Vechain-EasyA-Hackathon-Boston-Oct-2023/milestone-deliveries/Cypress-1.md

```diff
@@ -0,0 +1,26 @@
+
+* **PR Link:** [#207](https://github.com/vechain/grant-program/pull/207)
+
+* **Milestone Number:** Milestone 1
+
+Please provide a list of all deliverables of the milestone extracted from the initial application and a link to the deliverable itself. Ideally all links inside the below table should include a commit hash, which should be used for testing.
+
+| Number | Deliverable | Link | Notes |
+| ------------- | ------------- | ------------- |------------- |
+| 1. | Research |[Research Presentation](https://www.canva.com/design/DAF6cfFuufM/vcdMTYG8sdO6qLO6cJ7nWA/edit)| All consumer data and research available here, includes survey respondent data + desired workflow. Slides 1-7| 
+| 2.  | Requirements |[Requirements Document](https://docs.google.com/document/d/1iFZB31XT41khzqHYUzkxCHCL0ov-M5TMTjD0i8L0lE0/edit?usp=sharing)| Clear and concise requirements for blockchain integration, product tracking, transparent sustainability metrics, and technical implementation|
+| 3.  | Outreach |[Research Presentation](https://www.canva.com/design/DAF6cfFuufM/vcdMTYG8sdO6qLO6cJ7nWA/edit)| Contacted potential investors/partners such as Uber, BLUEBIKES, MBTA to see if a partnership is possible. Slides 11-14|
+| 4.  | Designs |[Competitive Analysis + UX/UI Details](https://www.figma.com/file/xYG6HZh7j5H0acJYD8oKcr/Cypress-Competitive-Analysis?type=whiteboard&node-id=0-1&t=PGb3V4NISAxEFM49-0) | Competitive analysis performed and refined the current UI/UX system based on market research and necessary requirements set out by consumers and brands.| 
+| 5.  | Backend Development |[Images located here](https://docs.google.com/document/d/1G2ePQPoRpvChe3VCsnSWnrCHJGOV9wLnr72aPklMcrU/edit) + [Github Repo for frontend located here, contains backend Web3Auth integration](https://github.com/cypress-labs/leegal)| Created custom user storage using Firebase Firestore, created Web3Auth Connection to Vechain Testnet| 
+| 6.  | Backend |[Go-based backend here](https://github.com/cypress-labs/backend/tree/main)| Set up the Go-based backend environment and version control system.| 
+| 7.  | Integration |[Demo Here](https://drive.google.com/file/d/17fu7obses7gZM2kR1pGqfb1ovSWXJpaY/view) |  This demo shows API endpoints are set up and tested. Integrates backend + smart contract + React Native frontend | 
+| 8.  | Frontend |[Demo Here](https://drive.google.com/file/d/17fu7obses7gZM2kR1pGqfb1ovSWXJpaY/view) + [Repo containing React Native frontend code here](https://github.com/cypress-labs/leegal/tree/main/cypress) + [Figma file containing prototype and added UX/UI pages here](https://www.figma.com/file/llEhoaZHO4GR2qSks42yjW/Cypress?type=design&node-id=26:726&mode=design&t=Np7KkiHSMdNx9ojP-1)| Frontend contains skeleton code for developing the garden management system. Figma contains hi-fidelity versions of what the garden management system will look like and improvements to the social system.| 
```

Comment:
In the initial application, Milestone 1.8 was described as the following
```
- Develop the garden management system, enabling users to plant, grow, and harvest rewards.
```
Which is different in this table, is the content changed?

Reply:
What we meant is that we've prototyped out what the garden management system will look like in our Figma (can click prototype to go through entire user workflow), and also worked out what this management system would look like once implemented.

I understand that our wording says "enabling users to plant, grow, and harvest rewards" but we apologize for not being clear - that is a goal that would be massively difficult to accomplish in the context of our React Native Frontend in the timeframe we set for Milestone 1.  What we meant is that we've prototyped out what the garden management system will look like in our Figma (can click prototype to go through entire user workflow), and also worked out what this management system would look like once implemented. This is also functionality we've enabled on our smart contract.

The brunt of our frontend development will occur next milestone.

Reply:
Hi, updated onto delivery table and freshly committed!